src/test/java/org/sonar/squidbridge/annotations/AnnotationBasedRulesDefinitionTest.java

```diff
@@ -155,29 +154,26 @@ public void class_with_nosqale_annotation() throws Exception {
     class RuleClass {
     }
 
-    Repository repository = buildRepository(true, RuleClass.class);
+    Repository repository = buildRepository(RuleClass.class);
     assertThat(repository.rules()).hasSize(1);
   }
 
   @Test
   public void class_with_sqale_constant_remediation() throws Exception {
 
     @Rule(key = "key1", name = "name1", description = "description1")
-    @SqaleSubCharacteristic(SubCharacteristics.CPU_EFFICIENCY)
```

Comment:
Shouldn't we keep at least one test to assert that no subcharacteristic is saved even when we use the annotation?

Reply:
Added a test.